socorro/external/crashstorage_base.py

```diff
@@ -584,26 +563,14 @@ def save_processed(self, processed_crash):
     def save_raw_and_processed(self, raw_crash, dump, processed_crash, crash_id):
         storage_exception = PolyStorageError()
 
-        # Later we're going to need to clone this per every crash storage
-        # in the loop. But, to save time, before we do that, convert the
-        # processed crash which is a SocorroDotDict into a pure python
-        # dict which we can more easily copy.deepcopy() operate on.
-        processed_crash_as_dict = socorrodotdict_to_dict(processed_crash)
-        raw_crash_as_dict = socorrodotdict_to_dict(raw_crash)
-
```

Comment:
I changed the architecture so `raw_crash` and `processed_crash` are dicts now. So we don't need to convert them or do any shenanigans with copying.